parsec/backend/cli/sequester.py

```diff
@@ -1,18 +1,19 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
 from __future__ import annotations
 
-from uuid import uuid4
-from base64 import b64encode, b64decode
-from async_generator import asynccontextmanager
-import textwrap
 import attr
 import click
-from parsec._parsec import DateTime
-from typing import Optional, Dict, List, Tuple
+import oscrypto
+import textwrap
+from oscrypto.asymmetric import PrivateKey
```

Comment:
Do we plan to replace `oscrypto` by the Rust impl ?

Reply:
@TimeEngineer up

Reply:
Do you think we can just replace ?

Reply:
I don't know that wasn't the question :shrug: 
More like if we want to restrain the usage of crypo to the crate `libparsec-crypto`

Reply:
We should

Reply:
I've created an issue for that #3327